README.rst

```diff
@@ -615,6 +615,11 @@ Choose from the list of available fixers:
                         closing semicolon are
                         prohibited.
 
+* **native_function_casing** [contrib]
```

Comment:
@keradus the tests should pass because `strlen` is the correct case ;)
maybe you want tests with overriden `StrLen` or something like this?

then it would not pass. but the change would not change code behavior
(https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1659#issuecomment-169722526)